packages/remix-edge-adapter/README.md

```diff
@@ -0,0 +1,3 @@
+## Remix Adapter for Netlify
```

Comment:
Mind having a peek at this @stephmarie17? It's to the point, but just want to make sure about the wording.

Reply:
LGTM!